azuredeploy-large-ha.json

```diff
@@ -1,65 +1,72 @@
-            "properties": {
-                "mode": "Incremental",
-                "parameters": {
-                    "_artifactsLocation":           { "value": "[parameters('_artifactsLocation')]" },
-                    "_artifactsLocationSasToken":   { "value": "[parameters('_artifactsLocationSasToken')]" },
-                    "redisDeploySwitch":            { "value": true },
-                    "sshPublicKey":                 { "value": "[parameters('sshPublicKey')]" },
-                    "autoscaleVmCountMax":          { "value": 20 },
-		            "autoscaleVmSku":		        { "value": "Standard_DS3_v2" },
-                    "searchType":                   { "value": "elastic" },
-                    "mysqlPgresVcores":             { "value": 16 },
-                    "mysqlPgresStgSizeGB":          { "value": 512 },
-                    "fileServerType":               { "value": "azurefiles" },
-                    "fileServerDiskSize":           { "value": 1024 },
-                    "storageAccountType":           { "value": "Premium_LRS" },
-                    "loadBalancerSku":              { "value": "Standard" },
-                    "OSDiskSizeInGB":               { "value": 1024 }
-                },
-                "templateLink": {
-                    "uri": "[concat(parameters('_artifactsLocation'), 'azuredeploy.json', parameters('_artifactsLocationSasToken'))]"
-                }
-            }
-        }
-    ],
-    "outputs": {
-        "mainTemplateOutputs": {
-            "type": "object",
-            "value": "[reference('mainTemplate').outputs]"
-        }
-    },
-    "variables": {
-        "documentation01": "This wrapper template calls the main-template with pre-defined configs for large size workloads (with high availability) and the only required parameter (sshPublicKey).",
-        "documentation02": "For the expected small-to-mid size workloads, other parameters are fixed in this tempalte and overriden as above."
-    }
+	"$schema": "https://schema.management.azure.com/schemas/2015-01-01/deploymentTemplate.json#",
+	"contentVersion": "1.0.0.0",
+	"parameters": {
+		"_artifactsLocation": {
+			"type": "string",
+			"metadata": {
+				"description": "The base URI where artifacts required by this template are located. When the template is deployed using the accompanying scripts, a private location in the subscription will be used and this value will be automatically generated."
+			},
+			"defaultValue": "https://raw.githubusercontent.com/Azure/Moodle/master/"
+		},
+		"_artifactsLocationSasToken": {
+			"type": "securestring",
+			"metadata": {
+				"description": "The sasToken required to access _artifactsLocation.  When the template is deployed using the accompanying scripts, a sasToken will be automatically generated."
+			},
+			"defaultValue": ""
+		},
+		"sshPublicKey": {
+			"metadata": {
+				"description": "ssh public key"
+			},
+			"type": "string"
+		},
+		"siteURL": {
```

Comment:
LGTM, apparently files were changed from tabs to spaces and added siteURL in all templates to make sure one specifies the domain for their moodle setup, which if left blank will be assined by Azure.